assets/models/system/salesforce-blip2-opt-2-7b-vqa/description.md

```diff
@@ -44,6 +44,7 @@ Note:
 ]
 ```
 
-#### Model inference: Text for the sample image
+#### Model inference - visual question answering
+For a sample image and text prompt "what are people doing? Answer: ".
```

Comment:
the other vqa example does not have "Answer:"

Reply:
only BLIP2 requires "Answer: "